console2/src/components/organisms/ProcessActivity/index.tsx

```diff
@@ -131,7 +131,15 @@ const ProcessActivity = (props: ExternalProps) => {
         setProcess(process);
 
         if (process.parentInstanceId && !rootProcessRef.current) {
-            rootProcessRef.current = await apiGetRoot(process.parentInstanceId);
+            apiGetRoot(process.parentInstanceId)
+                .catch(() => {
+                    rootProcessRef.current = null;
```

Comment:
Is it safe to set this to `process` (the current process) to avoid re-doing, and failing, this lookup each refresh?

Reply:
I don't _think_ so....since this `rootProcessRef` is used for process restart functionality

![Screenshot 2025-03-12 at 8 15 06 AM](https://github.com/user-attachments/assets/3861d40d-16a2-4609-89f1-393e94344759)

Reply:
Maybe? It will be the same process instance anyway. Should not matter when we load it.